packages/mattermost-redux/src/actions/general.ts

```diff
@@ -148,15 +148,6 @@ export function setStoreFromLocalData(data: { token: string; url: string }): Act
     };
 }
 
-export function getSupportedTimezones() {
```

Comment:
Are we going to remove the corresponding API route for this, or is that going to hang around for the time being? 6.0 could be an excuse to remove the Client4 method for it as well.

Reply:
That was mentioned but could be an idea. I'll check it out

Reply:
I removed the route from Client4 but the server endpoint is still there

Reply:
That works for me. The mobile app also uses these timezones, so we'd have to wait for old versions of the mobile app to stop using it before it could be removed from the server.